frontend/public/components/RBAC/bindings.jsx

```diff
@@ -131,30 +162,35 @@ export const RoleLink = ({binding}) => {
   return <ResourceLink kind={kind} name={binding.roleRef.name} namespace={ns} />;
 };
 
-const Row = ({obj: binding}) => <ResourceRow obj={binding}>
-  <div className="col-md-3 col-sm-4 col-xs-6">
-    <ResourceLink kind={bindingKind(binding)} name={binding.metadata.name} namespace={binding.metadata.namespace} className="co-resource-item__resource-name" />
-  </div>
-  <div className="col-md-3 col-sm-4 hidden-xs co-break-word">
-    <RoleLink binding={binding} />
-  </div>
-  <div className="col-md-2 hidden-sm hidden-xs co-break-word">
-    {binding.subject.kind}
-  </div>
-  <div className="col-md-2 hidden-sm hidden-xs co-break-word">
-    {binding.subject.name}
-  </div>
-  <div className="col-md-2 col-sm-4 col-xs-6 co-break-word">
-    {binding.metadata.namespace ? <ResourceLink kind="Namespace" name={binding.metadata.namespace} /> : 'all'}
-  </div>
-  <div className="dropdown-kebab-pf">
-    <BindingKebab binding={binding} />
-  </div>
-</ResourceRow>;
+const RoleBindingsTableRow = ({obj: binding, index, key, style}) => {
```

Comment:
It's unfortunate we have to handle three extra props in every table.

Reply:
I would try to use Object rest here.

```suggestion
const RoleBindingsTableRow = ({obj: binding, ...vrProps}) => {
```

and then

```jsx
<Vr {...vrProps}>
```

We'd have to resolve the key/trKey difference, though.